content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst

```diff
@@ -1,267 +1,315 @@
-as well.
+.. note::
+   Sometimes referred to as a *call for tender*, this process is primarily used by organizations in
+   the public sector, who are legally bound to use it when making a purchase. However, private
+   companies can also use alternative |RfQs| to spend money efficiently.
 
-.. seealso::
-   :doc:`blanket_orders`
+Configuration
+=============
 
-Configure purchase agreement settings
-=====================================
+To create alternative |RfQs|, the *Purchase Agreements* feature **must** be enabled in the
+*Purchase* app settings. To enable the feature, navigate to :menuselection:`Purchase app -->
+Configuration --> Settings`. Under the :guilabel:`Orders` section, click the checkbox for
+:guilabel:`Purchase Agreements`.
 
-To create alternative :abbr:`RFQs (Requests for Quotation)` directly from a quotation, the *Purchase
-Agreements* feature first needs to be enabled in the settings of the *Purchase* app. To do this, go
-to :menuselection:`Purchase --> Configuration --> Settings`, and under the :guilabel:`Orders`
-section, click the checkbox next to :guilabel:`Purchase Agreements`. Doing so will enable the
-ability to create alternative :abbr:`RFQs (Requests for Quotation)`, as well as the ability to
-create *blanket orders*.
+Then, click :guilabel:`Save` to apply the change.
 
-.. image:: calls_for_tenders/calls-for-tenders-settings-page.png
+Doing so enables the ability to create alternative |RfQs|, as well as *blanket orders*.
+
+.. image:: calls_for_tenders/calls-for-tenders-enabled-setting.png
    :align: center
    :alt: Purchase Agreements enabled in the Purchase app settings.
 
 .. tip::
-   To save time on a *call for tender*, custom vendors, prices, and delivery lead times can be set
-   in the :guilabel:`Purchase` tab on a product form. To do so, navigate to
-   :menuselection:`Purchase --> Products --> Products`, and select a product to edit. From the
-   product form, click the :guilabel:`Purchase tab`, then click :guilabel:`Add a line`. From the
-   drop-down menu, choose a vendor to set under the :guilabel:`Vendor` column, and set a
-   :guilabel:`Price` and :guilabel:`Delivery Lead Time` if desired. Clicking the
-   :guilabel:`additional options (two-dots)` icon provides additional visibility options to add to
+   To save time creating alternative |RfQs|, custom vendors, prices, and delivery lead times can be
+   set in the :guilabel:`Purchase` tab of a product form.
+
+   To do that, navigate to :menuselection:`Purchase app --> Products --> Products`, and select a
+   product to edit. On the product form, click the :guilabel:`Purchase` tab, and click
+   :guilabel:`Add a line`.
+
+   From the drop-down menu, choose a vendor to set under the :guilabel:`Vendor` column, then set a
+   :guilabel:`Price` and :guilabel:`Delivery Lead Time`, if necessary. Clicking the
+   :guilabel:`(optional columns drop-down toggle)` icon provides additional column options to add to
```

Comment:
perhaps it would helpful to always mention where the  user can find this icon...

Reply:
Added clarification for where to find this icon. Suggestion implemented ✅